index.html

```diff
@@ -79,7 +82,7 @@
 
                         <h1 class="brand-heading">Breeze '16</h1>
                         <p class="intro-text">Shiv Nadar University</p>
-                        <a href="#contact" class="btn btn-circle page-scroll">
```

Comment:
Scroll btn link for Contact section is already made at the end of About Us section. The btn you pointed out is only to reach the next immediate section wiz. About Us.